recipes/flask-jwt-extended/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "Flask-JWT-Extended" %}
+{% set version = "3.3.1" %}
+{% set sha256 = "fe25451a67641ac3007ebbde575e35dfbda7f95b95434b65ff10e3d37f2986b7" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - flask >=0.12.2
```

Comment:
The version requirements for `flask` and `pyjwt` are not specified in the `setup.py`. I just wanted to make sure they are correct (I can see versions listed in the `requirements.txt`, but I'm not sure if those are strict).

Reply:
So I will let it without version requirement since setup.py do not have any.